safehttp/incoming_request.go

```diff
@@ -96,13 +97,13 @@ func (r *IncomingRequest) PostForm() (*Form, error) {
 // used when the user expects a POST request with the Content-Type: multipart/form-data header.
 func (r *IncomingRequest) MultipartForm(maxMemory int64) (*MultipartForm, error) {
 	var err error
-	// Ensures no more than 32 MB are stored in memory when a form file is
-	// passed as part of the request. If this is bigger than 32 MB, the rest
-	// will be stored on disk.
-	const defaultMaxMemory = 32 << 20
 	r.multipartParseOnce.Do(func() {
-		if r.req.Method != "POST" && r.req.Method != "PATCH" && r.req.Method != "PUT" {
-			err = fmt.Errorf("got request method %s, want POST/PATCH/PUT", r.req.Method)
+		// Ensures no more than 32 MB are stored in memory when a form file is
+		// passed as part of the request. If this is bigger than 32 MB, the rest
+		// will be stored on disk.
+		const defaultMaxMemory = 32 << 20
```

Comment:
If you want you can move this closer to its use, on line 113

Reply:
This is getting removed in #135 so i'm not gonna bother.